src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-details/rgw-multisite-details.component.ts

```diff
@@ -245,10 +255,17 @@ export class RgwMultisiteDetailsComponent implements OnDestroy, OnInit {
       click: () => this.openExportModal(),
       disable: () => this.getDisableExport()
     };
+    const setupMultisiteReplication: CdTableAction = {
+      permission: 'create',
+      icon: Icons.wrench,
+      name: this.actionLabels.SETUP_MULTISITE_REPLICATION,
+      click: () => this.openMultisiteSetupWizard()
+    };
```

Comment:
btw, you can directly assign this to the table actions? why do we need a separate constant?

Reply:
Done